bigtable/bttest/inmem.go

```diff
@@ -972,10 +968,6 @@ func (s *server) CheckAndMutateRow(ctx context.Context, req *btpb.CheckAndMutate
 	return res, nil
 }
 
-func (s *server) PingAndWarm(ctx context.Context, req *btpb.PingAndWarmRequest) (*btpb.PingAndWarmResponse, error) {
```

Comment:
Why remove `PingAndWarm()`?

Reply:
Erm, not sure how it ended up being removed, it's been a while but I'm sure I only touched lines 748-751 in this file.

Let me also rebase this branch.

Reply:
well, git history says I removed it...

Reply:
@telpirion reverted that change in https://github.com/googleapis/google-cloud-go/pull/5973/commits/60042f3c1898726f0a75af68c895f017d1dab17b, please let me know if there are any other comments to address, thanks in advance!